etc/spack/defaults/packages.yaml

```
@@ -59,7 +59,7 @@ packages:
       qmake: [qt-base, qt]
       rpc: [libtirpc]
       scalapack: [netlib-scalapack, amdscalapack]
-      sycl: [hipsycl]
+      sycl: [hipsycl, adaptivecpp]
```

In your PR you changed the packages default to sycl: [hipsycl, adaptivecpp], as a result if i'm right most should default to hipsycl instead of adaptivecpp. Shouldn't we invert the two ?
Especially since people building against hipsycl in spack will build agains 24.x.x do that should change basically nothing but avoid defaulting to the old one

---

Thanks, that's a good point.

```suggestion
      sycl: [adaptivecpp, hipsycl]
```